cmd/push/imagebundle/image_bundle.go

```diff
@@ -47,22 +48,43 @@ func NewCommand(out output.Output) *cobra.Command {
 			cleaner.AddCleanupFn(func() { _ = os.RemoveAll(tempDir) })
 			out.EndOperation(true)
 
-			out.StartOperation("Unarchiving image bundle")
-			err = archive.UnarchiveToDirectory(imageBundleFile, tempDir)
-			if err != nil {
-				out.EndOperation(false)
-				return fmt.Errorf("failed to unarchive image bundle: %w", err)
-			}
-			out.EndOperation(true)
+			sort.Strings(imageBundleFiles)
 
-			out.StartOperation("Parsing image bundle config")
-			cfg, err := config.ParseImagesConfigFile(filepath.Join(tempDir, "images.yaml"))
-			if err != nil {
-				out.EndOperation(false)
-				return err
+			var cfg config.ImagesConfig
+
+			// Just in case users specify the same bundle twice, keep a track of
+			// files that have been extracted already so we only extract each of them once.
+			extractedBundles := make(map[string]struct{}, len(imageBundleFiles))
+
+			for _, imageBundleFile := range imageBundleFiles {
```

Comment:
This looks the same as https://github.com/mesosphere/mindthegap/pull/167/files#diff-54b016241b0fa933e5681a296d26053d15b23f9ad15f8d4c09c050533e80889aR55, is there an opportunity to pull it out into a common function?

Reply:
Done, thank you!

Reply:
I dont see it 🤔  https://github.com/mesosphere/mindthegap/pull/169/files#diff-54b016241b0fa933e5681a296d26053d15b23f9ad15f8d4c09c050533e80889aR55 and https://github.com/mesosphere/mindthegap/pull/169/files#diff-b8bf632cc190cd3f09692e0cdd5535c70cc065db145a9d998b4a33c7a6ce339eR59